### ISSUE TITLE
Update Links and Formatting

### ISSUE THREAD

**Issue by erinmagennis** [kylehalleman/covid19inwi#15]
Rec: make the "Covid-19 in Wisconsin" header clickable back to the homepage 
 
Specific page recs: 
Milwaukee Responds to Covid-19  - 
1.) the monotony of the layout of the words on this page makes it very overwhelming to look at and understand
Rec: have drop downs for each section so you click on the heading and then all of the bullets show up
2.) if you click on almost any of the other top links it creates a 404 error and the only way to get out is to remove /old/ from the url  

General information and data - broken link 

How to get help & How to help - more manageable but still monotonous and overwhelming 

How to contribute to this resource - create more welcoming/inviting language to have people contribute on Github

Questions to consider: 
- can there be some pictures/icons/something to make the UI a little better? 
- is this ada compliant? 
- could you add a search bar to make all of the content more manageable/findable?

**Comment by kylehalleman** [comment]
It looks like you found a bug. The Milwaukee Responds to COVID-19 page was the old site, which included everything that is now split across multiple pages. I just removed it from the nav so it shouldn't show up anymore and should mean you won't encounter any 404s anymore.

I would err against the use of accordions for sections, as it introduces usability issues. The table of contents for each page makes navigating to the section one wants to get to easier. I think we could and should add "jump to top" links that take users back to the table of contents though. This is an accessible and simple way to implement the accordion type of functionality you're talking about.

Other answers:
* UI improvements can be discussed in #14... I have no specific changes at this time but am open to suggestions!
* This site uses semantic HTML and passes WCAG 2.1 for color contrast and reflow (and other categories although I haven't specifically tested against them). I have tested it with a screen reader and aXe tools and it passes with no issues.
* I will look into adding a search of some sort.

Thank you for the comments and review!

**Comment by Aycrazy** [comment]
Added link to the header for ppl to return to the landing page that way too

**Comment by erinmagennis** [comment]
https://socmilwaukee.org/index/coronavirus-information/
^ this layout is a lot more reader friendly - might be simply due to spacing and good size proportions of headers to body

**Comment by erinmagennis** [comment]
Additional thoughts: 
- on the sides (or somewhere prominent) of the webpage have the main symptoms + the main things you should do (wash hands, not rush to the er right away, stay home) 
- this could be a good resource from Mayo Clinic to help people decide what they should do (dr or not) - https://www.mayoclinic.org/covid-19-self-assessment-tool
- make it more clear that the top links take you to a whole new page of resources 
(the 3 links at the top should be formatted differently/more prominently - because there are a bunch of great resources on them)

**Comment by kylehalleman** [comment]
Thanks, I'll work on these updates today.

**Comment by kylehalleman** [comment]
Updated main nav links at top to stand out more, added context of "I want to..." as well

**Comment by erinmagennis** [comment]
Is this linked somewhere? Seems useful. https://mcoem.maps.arcgis.com/apps/opsdashboard/index.html#/018eedbe075046779b8062b5fe1055bf

**Comment by kylehalleman** [comment]
Yes, it's linked under General Information > Official Sources > "Milwaukee County COVID-19 Dashboard and news releases"

edit: There's also the same dashboard but for all of Wisconsin linked under Data.
